erpnext/stock/stock_ledger.py

```diff
@@ -202,8 +202,7 @@ def get_sle_against_current_voucher(self):
 			where
 				item_code = %(item_code)s
 				and warehouse = %(warehouse)s
-				and voucher_type = %(voucher_type)s
-				and voucher_no = %(voucher_no)s
+				and timestamp(posting_date, posting_time) = timestamp(%(posting_date)s, %(posting_time)s)
```

Comment:
we need to avoid milliseconds
```suggestion
				and timestamp(posting_date, time_format(posting_time, '%H:%i:%s')) = timestamp(%(posting_date)s, time_format(%(posting_time)s, '%H:%i:%s'))
```